### ISSUE TITLE
Is Repo aligned to Master?

### ISSUE THREAD

**Issue by carloBarazzetta** [Templarian/MaterialDesign-Font#6]
I'm downloading https://raw.githubusercontent.com/Templarian/MaterialDesign/master/meta.json definition with 6628 icons, but this font (cheatsheet.html) contains only 6624 icons: is not yet aligned with master repository?
Thanks.

**Comment by Templarian** [comment]
tl;dr: meta.json is a realtime sync while fonts only include icons associated with a release version.

The `meta.json` is near real time in the main repo with the database. All font generation requires a codepoint to be assigned to show up in the font (think of this an an index number represented as hex value). For a codepoint to be assigned it must be associated to a release tag in the database. Icon -> Version + Codepoint. This also lets us associate a version to when an icon was added to a release. Not all the contributors really do this step since it's not really required until the we cut a release.

You can use the `@mdi/font-build` package. It takes a folder of `/svg` a `font-build.json` file and a `meta.json` to make the fonts.

Learn More: https://www.npmjs.com/package/@mdi/font-build + `@mdi/svg`

**Comment by carloBarazzetta** [comment]
Thank you very much!
